### ISSUE TITLE
Improve "BUILD START" logging

### ISSUE THREAD

**Issue by frol** [conda/conda-build#875]
First of all, this `BUILD START`/`BUILD END` looks like debug output:

```
Removing old build environment
Removing old work directory
BUILD START: mypackage--py27_0
Fetching package metadata: ..............
Solving package specifications: .........
```

See how text around `BUILD START` is human-friendly.

Second, using remote git source, you don't have an actual version and build number yet, so what is the point in confusing developers?! Just print the information you have, i.e. package name and python version, in a human-friendly format, and later, once you gathered all the information, print another message. This confusion seems to pop up in every git-based recipe issues: #305, #357, #764...

**Comment by stuarteberg** [comment]
Some quick background info:

The `BUILD_START` line used to be even worse -- it would use arbitrary values for `GIT_DESCRIBE_TAG`, etc. (whatever the previously built package defined)!  After #758, those variables are empty until the git variables can be accurately determined.

As you point out, the consequence of this is that the `BUILD_START` line now looks "broken"  because it is obviously missing information.  (Previously, it looked plausible, but happened to be totally wrong.)

>Just print the information you have, i.e. package name

Agreed.  Someone should implement a pull request that changes the `BUILD_START` line so that only the package name is printed, not the full `name`-`version`-`build_string`.  That would be much clearer.  Any takers?

**Comment by groutr** [comment]
https://github.com/conda/conda-build/pull/885

**Comment by msarahan** [comment]
Closed by #923
